.circleci/config.yml

```diff
@@ -76,7 +76,7 @@ jobs:
           detect-login-type: true
 
   test-orb-golang:
-    executor: golang
+    executor: go
```

Comment:
If we are changing the argument we are passing here, we also need to change the executor's name from `golang` to `go`: https://github.com/CircleCI-Public/azure-cli-orb/blob/master/.circleci/config.yml#L19